README.md

```diff
@@ -1,6 +1,9 @@
 meta-search-engine
 ==================
 
+[![Build Status](https://travis-ci.org/schul-cloud/meta-search-engine.svg?branch=master)](https://travis-ci.org/schul-cloud/meta-search-engine)
+[![Docker Build Status](https://img.shields.io/docker/build/jrottenberg/ffmpeg.svg)](https://hub.docker.com/r/schulcloud/meta-search-engine/builds/)
```

Comment:
In the docker line, you do not use the image from schulcloud/meta-search-engine but from jrottenberg/ffmpeg
Apart from this, this is what we are looking for.

Reply:
All the things you did "right" which could have gone wrong:
- use markdown, not html
- find out how to include images
- find the right images to include 1x
- use links 2x